src/Microsoft.Azure.WebJobs.Logging/Entities/InstanceTableEntity.cs

```diff
@@ -90,9 +89,14 @@ internal static string RowKeyFunctionInstanceId(Guid functionInstanceId)
 
         public string TriggerReason { get; set; }
 
-        public DateTimeOffset StartTime { get; set; }
+        public DateTime StartTime { get; set; }
```

Comment:
I verified this is not a breaking change. I did a test to write in one format, read in the other, and vice versa.  Specifically, Azure Tables only supports EDM.DateTime, and the DateTimeOffset is just client-side sugar. They both write the same underlying value to the table and the server doesn't know the difference. We should be using UTC DateTime everywhere rather than DateTimeOffset.